lib/xray_creator.py

```diff
@@ -42,12 +42,8 @@ def _initialize_books(self):
 
         self._books = []
         for book_id in self._book_ids:
-            book = Book(self._db, book_id, self._gConnection, self._aConnection, formats=self._formats,
-                send_to_device=self._send_to_device, create_xray=self._create_xray, proxy=self._proxy,
-                https_address=self._https_address, https_port=self._https_port)
-            self._books.append(book)
-            self._gConnection = book._gConnection
-            self._aConnection = book._aConnection
+            self._books.append(Book(self._db, book_id, self._gConnection, self._aConnection, formats=self._formats,
+                send_to_device=self._send_to_device, create_xray=self._create_xray))
         
         self._total_not_failing = 0
         book_lookup = {}
```

Comment:
I thought i removed this but i think something happened when i went to
merge or something -- i'm adding it back in

On Tue, Jun 28, 2016 at 5:18 AM Anthony Toole <notifications@github.com>
wrote:

> In lib/xray_creator.py
> <https://github.com/szarroug3/X-Ray_Calibre_Plugin/pull/54#discussion_r68731772>
> :
>
> > @@ -42,12 +42,8 @@ def _initialize_books(self):
> >
> >          self._books = []
> >          for book_id in self._book_ids:
> > -            book = Book(self._db, book_id, self._gConnection, self._aConnection, formats=self._formats,
> > -                send_to_device=self._send_to_device, create_xray=self._create_xray, proxy=self._proxy,
> > -                https_address=self._https_address, https_port=self._https_port)
> > -            self._books.append(book)
> > -            self._gConnection = book._gConnection
> > -            self._aConnection = book._aConnection
> > +            self._books.append(Book(self._db, book_id, self._gConnection, self._aConnection, formats=self._formats,
> > +                send_to_device=self._send_to_device, create_xray=self._create_xray))
> >
> >          self._total_not_failing = 0
> >          book_lookup = {}
>
> The cleanup you added looks good, but you didn't do it all (you got rid of
> the unnecessary return values, but eg. we are still passing through
> gConnection when calling book.create_xray_event - when it could be cached
> at init time.
>
> You might have had a reason for not removing that though, so just calling
> it out.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/szarroug3/X-Ray_Calibre_Plugin/pull/54/files/89c7d995442049dfa337e6394eb310d53c334dbd#r68731772>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIEjUVCWX54yQiTinNRFSrw9Q7iG9U7Bks5qQPUCgaJpZM4I90sF>
> .
>